app/controllers/spaces_controller.rb

```diff
@@ -9,6 +9,9 @@ def create
 
   def show
     @spaces = Space.all
+    @users = User.all
```

Comment:
I don't think we should be doing this - if we want to pull user data into this view we should be running a join to gather relevant records rather than just .all on all tables